src/cr/cube/matrix/assembler.py

```diff
@@ -575,36 +575,18 @@ def table_margin(self):
 
         This value has four distinct forms, depending on the slice dimensions:
 
-            * MR_X_MR - 2D ndarray with a distinct table-margin value per cell.
-            * MR_X - 1D ndarray of value per *row* when only rows dimension is MR.
-            * X_MR - 1D ndarray of value per *column* when only columns dimension is MR.
-            * CAT_X_CAT - scalar float value when slice has no MR dimension.
-
+            * CAT_X_CAT - scalar float value when slice has no ARRAY dimension.
+            * ARRAY_X - 1D ndarray of value per *row* when only rows dimension is ARRAY.
+            * X_ARRAY - 1D ndarray of value per *column* when only column is ARRAY.
+            * ARRAY_X_ARRAY - 2D ndarray with a distinct table-margin value per cell.
```

Comment:
My goal has been to minimize external interface changes to try to reduce scope. I agree that ideally it would be changed everywhere, but this would also mean altering `_Stripe` to be truly everywhere.

Your suggestion at the end doesn't preserve the interface because `self._assembler.table_weighted_base` is always either a scalar or None, whereas `table_margin` has always been the unknown dimension thing.